base/floatfuncs.jl

```diff
@@ -274,6 +274,15 @@ function isapprox(x::Number, y::Number; atol::Real=0, rtol::Real=rtoldefault(x,y
     x == y || (isfinite(x) && isfinite(y) && abs(x-y) <= max(atol, rtol*max(abs(x), abs(y)))) || (nans && isnan(x) && isnan(y))
 end
 
+"""
+    isapprox(x; kwargs...) / ≈(x; kwargs...)
+
+Create a function that compares its argument to x using ≈, i.e. a function equivalent to y -> y ≈ x.
+
+The keyword arguments supported here are the same as those in [`isapprox`](@ref).
```

Comment:
The Documenter ref there is self-referential. Probably doesn't matter a whole lot, but you could have it point to the other method specifically.

Reply:
How do you do that - with the method signature? So something like `[\`isapprox(1, 2)\`]`?

Reply:
I'm not 100% on the syntax, but I think it'd be something like
```markdown
[`isapprox`](@ref Base.isapprox(::Any, ::Any))
```
You'd probably want to build the documentation locally to verify.

Reply:
Or just say "The keyword arguments supported are the same as those in 2-argument `isapprox`."